app/src/main/res/values/strings.xml

```diff
@@ -341,6 +341,26 @@
     <string name="reduce_cookie_banner_details_panel_description_off_for_site">%1$s will clear this site’s cookies and refresh the page. Clearing all cookies may sign you out or empty shopping carts.</string>
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is on for a site, this is shown as part of the cookie banner panel in the toolbar. The first and second parameter are the application name -->
     <string name="reduce_cookie_banner_details_panel_description_on_for_site">%1$s can try to automatically reject cookie requests. If a reject option isn’t available, %2$s may accept all cookies to dismiss the banner.</string>
+    <!-- Title text for the onboarding dialog use on the control branch of the experiment to determine which context users engaged the most-->
+    <string name="reduce_cookie_banner_control_experiment_dialog_title">Cookie banners begone!</string>
+    <!-- Body text for the onboarding dialog use on the control branch of the experiment to determine which context users engaged the most-->
```

Comment:
```suggestion
    <!-- Body text for the onboarding dialog use on the control branch of the experiment to determine which context users engaged the most -->
```
Could you add a space before `-->` for all of these?

Reply:
Updated :) !